### PR TITLE
Issue/upgrade queues to spring3

### PR DESCRIPTION
**Pull request by LuisLainez**
Upgrade orkes-queues to Spring3 so it can work with the new Conductor Spring 3 version.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -11,8 +11,8 @@ plugins {
 
 ext {
     group = 'io.orkes.queue'
-    appVersion = '0.0.2-SNAPSHOT'
-    springBootVersion = '2.5.6'
+    appVersion = '0.0.5-SNAPSHOT'
+    springBootVersion = '3.1.5'
```
Comment (LuisLainez) [inline build.gradle:15]: Note to reviewers - I don't know how this is going to look on release time.
We are in version 1.0.6 https://mvnrepository.com/artifact/io.orkes.queues/orkes-redis-queues - I don't know if this number will impact that versioning.

File: orkes-conductor-queues/src/main/java/io/orkes/conductor/queue/config/RedisQueueConfiguration.java [hunk 2]
```diff
@@ -12,31 +12,30 @@
...
-@Configuration
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@EnableAutoConfiguration
+@AutoConfiguration
 @Slf4j
+@Import({io.orkes.conductor.queue.config.QueueRedisProperties.class})
```
Comment (LuisLainez) [inline orkes-conductor-queues/src/main/java/io/orkes/conductor/queue/config/RedisQueueConfiguration.java:38]: I could not make it work with any other way of autowiring configuration.
Please feel free to make any changes if you have a more elegant way to add these properties

Comment (thombergs) [reply orkes-conductor-queues/src/main/java/io/orkes/conductor/queue/config/RedisQueueConfiguration.java:38]: I think the library is missing a clear concept of how to integrate with a Spring / Spring Boot application. Ideally, there is a module that contains the library features and a second module that provides a Spring auto-configuration that activates automatically when added as a dependency to a Spring project. This is not the case here. Instead, there are some `@Configuration` and `@ConfigurationProperties` classes but no clear guidance of how to activate them.

In the README, it says somewhere that you should add a `@ComponentScan("io.orkes.conductor")` to your code. Did you do that? 

Anyway, medium term I suggest to add a separate "*-spring" module with auto-configurations. Short term I suggest to provide clear guidance of how to integrate this library with a Spring Boot application.

Comment (LuisLainez) [reply orkes-conductor-queues/src/main/java/io/orkes/conductor/queue/config/RedisQueueConfiguration.java:38]: @ComponentScan("io.orkes.conductor")
This is done in the conductor repository - where we load the application library
`@ComponentScan(basePackages = {"com.netflix.conductor", "io.orkes.conductor"})
`

**Review by v1r3n — APPROVED** [review]

**Pull request merged**